### PR TITLE
feat - userContent And Playlists

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Add User Content and Playlists.
minor changes are rebased, 
Preview - https://video-lib-react-git-usercontent-and-playlists-vishnu-aithal.vercel.app/

### CODE REVIEW

**Review by MayankKumar10 — APPROVED** [review]
The code is clean nothing to suggest Looks good to me you can merge it.

File: src/utility-functions/userHandler.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
+import axios from "axios";
+
+export const loadUserData = async (
```
Comment (jalajcodes) [inline src/utility-functions/userHandler.js:3]: Appreciation: all user related things in one file

**Review by jalajcodes — APPROVED** [review]
Looks good. Code is clean and organized

**Comment by Vishnu-Aithal** [comment]
Thank You @MayankKumar10  and @jalajcodes for PR Review . Merging now

**Pull request merged**